test/char.carp

```diff
@@ -5,17 +5,21 @@
 
 (deftest test
   (assert-equal test
-		"a"
-  		&(str \a)
-		"str works with ASCII")
+    "Π"
+    &(str \u03a0)
+    "unicode literals work")
   (assert-equal test
-		"ñ"
-  		&(str \ñ)
-		"str works with Latin1")
+                "a"
+                &(str \a)
+                "str works with ASCII")
   (assert-equal test
-		"😲"
-  		&(str \😲)
-		"str works with Emoji")
+                "ñ"
+                &(str \ñ)
+                "str works with Latin1")
+  (assert-equal test
+                "😲"
+                &(str \😲)
+                "str works with Emoji")
```

Comment:
Should we add tests for the special cases too? `\backspace` etc.?

Reply:
Yeah. I was honestly just to lazy to figure out how to best test it, but that’s not an excuse! I’ll fix that tomorrow :)

Reply:
I added tests for all the literals!